sysutils/zsh-syntax-highlighting/Portfile

```diff
@@ -26,16 +25,14 @@ license             BSD
 platforms           any
 supported_archs     noarch
 
-checksums           sha256  5abb9814ed5fcfc9d393be4932adb86b39de3b324eee8ade2debe44530acd840 \
-                    rmd160  9df8771fb309f44f62a41fb0af65ddc34296d506 \
-                    size    133551
+checksums           rmd160  a05ac72e372c4012e5be7cb5d62031eebad2ad64 \
+                    sha256  0e329871928620b6f521bf366dfe8f8ddf4b03d97f8fa3eb6af25a26a6ed9b21 \
+                    size    155949
 
-depends_run         path:bin/zsh:zsh
+depends_run-append  path:bin/zsh:zsh
 
-use_configure       no
+test.run            yes
+test.args-append    ZSH="${prefix}/bin/zsh"
+test.target         test
 
-build { }
-
-destroot.args-append ZSH=${prefix}/bin/zsh PREFIX=$prefix
```

Comment:
why is the `ZSH="${prefix}/bin/zsh"` no longer needed here; that shouldn't be set by the `makefile` PG, whereas `PREFIX=$prefix` is set by the PortGroup

Reply:
`ZSH="${prefix}/bin/zsh"` is no longer required there since [the Makefile only uses it when testing](https://github.com/zsh-users/zsh-syntax-highlighting/blob/master/Makefile#L38-L49)